newscoop/library/Newscoop/Entity/Repository/ArticleRepository.php

```diff
@@ -68,6 +68,14 @@ public function getArticles($publication, $type = null, $language = null, $issue
                 ->setParameter('issue', $issue);
         }
 
+        if ($section) {
+            $countQueryBuilder->andWhere('a.sectionId = :section')
```

Comment:
this is duplication, $countQueryBuilder should be cloned in this method because you need to write the same code for two query builders